### ISSUE TITLE
Bug : create a ws in 7.0 throws error

### ISSUE THREAD

**Issue by CPASimUSante** [claroline/Distribution#565]
I tried to create a workspace, through the ws page /workspaces/new/form
I filled the name and code, but when i submit i have : 
```

Failed to create "/tmp/templates/default.zip": The possibility to register the same extension twice ("pagerfanta") is deprecated since version 1.23 and will be removed in Twig 2.0. Use proper PHP inheritance instead..
500 Internal Server Error - IOException

Stack Trace
in vendor/symfony/symfony/src/Symfony/Component/Filesystem/Filesystem.php at line 100   -
                if (!is_dir($dir)) {
                    // The directory was not created by a concurrent process. Let's throw an exception with a developer friendly error message if we have one
                    if ($error) {
                        throw new IOException(sprintf('Failed to create "%s": %s.', $dir, $error['message']), 0, null, $dir);
                    }
                    throw new IOException(sprintf('Failed to create "%s"', $dir), 0, null, $dir);
                }
at Filesystem ->mkdir ('/tmp/templates/default.zip') 
in vendor/claroline/distribution/main/core/Manager/WorkspaceManager.php at line 1132   + 
at WorkspaceManager ->getTemplateData (object(File), true) 
in vendor/claroline/distribution/main/core/Manager/TransferManager.php at line 169   + 
at TransferManager ->createWorkspace (object(Workspace), object(File)) 
in vendor/claroline/distribution/main/core/Manager/WorkspaceManager.php at line 172   + 
at WorkspaceManager ->create (object(Workspace), object(File)) 
in vendor/claroline/distribution/main/core/Controller/WorkspaceController.php at line 363   + 
at WorkspaceController ->createAction () 
at call_user_func_array (array(object(WorkspaceController), 'createAction'), array()) 
in app/bootstrap.php.cache at line 3109   + 
at HttpKernel ->handleRaw (object(Request), '1') 
in app/bootstrap.php.cache at line 3071   + 
at HttpKernel ->handle (object(Request), '1', true) 
in app/bootstrap.php.cache at line 3222   + 
at ContainerAwareHttpKernel ->handle (object(Request), '1', true) 
in app/bootstrap.php.cache at line 2444   + 
at Kernel ->handle (object(Request)) 
in web/app_dev.php at line 28   + 
Logs   - 1 error
INFO - Matched route "claro_workspace_create". 
```

Of course i did
`sudo rm -rf  app/cache*`
`sudo chmod -R 777 app/ files web`
to be sure.

**Comment by ngodfraind** [comment]
try chmod -R 0777 /tmp/template

**Comment by CPASimUSante** [comment]
it works ! :+1:
I should have checked this path.

**Comment by ngodfraind** [comment]
I think it's because this directory is created during the install. If you used a root account, permissions won't be ok anymore.

**Comment by ngodfraind** [comment]
Fixed by #580
